From e45e531aa955c0ea6c4fc3278c2695d2e7ff33f9 Mon Sep 17 00:00:00 2001
From: "junkai.li" <junkai.li@sky-data.cn>
Date: Fri, 2 Mar 2018 14:32:42 +0800
Subject: [PATCH] Modify H2O Dockerfile

---
 scheduler/docker/h2o/Dockerfile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scheduler/docker/h2o/Dockerfile b/scheduler/docker/h2o/Dockerfile
index 947b8d5..6a2ee8d 100644
--- a/scheduler/docker/h2o/Dockerfile
+++ b/scheduler/docker/h2o/Dockerfile
@@ -1,11 +1,11 @@
 # Pull base image
 # build docker image name : skydiscovery_py2.7_tf1.2.0_cpu:1.0
-FROM opsh2oai/h2o-deepwater:latest
+FROM 192.168.20.38/skyaxe/h2o_deepwater:3.0
 
 MAINTAINER WeiJia.Liu "weijia.liu@sky-data.cn"
 
 # Add run.sh and set entrypoint
-ADD run.sh /usr/local
+ADD pkgs/run.sh /usr/local
 RUN chmod a+x /usr/local/run.sh
 
-ENTRYPOINT ["/bin/bash", "/usr/local/run.sh"]
\ No newline at end of file
+ENTRYPOINT ["/bin/bash", "/usr/local/run.sh"]
-- 
1.8.3.1

